Paper bug fixes page

olijeffers0n opened the pull request:
This is the initial page for transferring the Github Issue to an actual page. Let me know any comments you have as You may want more of an explanation to why these were changed. Any maintainers may also have more insight.

the456gamer commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,52 @@
+
+:::danger[Sand and Gravity Blocks]
+
+**We will not and will never support sand and gravity block duping**. This is a form of resource duping and is **not** allowed.
```
Is allowed the right word?
Since server owners can do what they want, including installing plugins to bring the dupe back

olijeffers0n replied:
Probably not

lynxplay commented on docs/paper/admin/misc/paper-bug-fixes.md:
The main reason iirc why these options are in the unsupported setting is not really the lag or enabling them, it is more that we do not promise these to keep working down the line.

If upstream fixes this accidentally, or even mojang, we are not going out of our way to re-implement these settings.
If enabling them absolutely crashes your server, it is probably still a good idea to let us know lol.

olijeffers0n replied:
Just copied this from Aikars Description. Have added another sentence now

Machine-Maker commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,53 @@
+
+## Vanilla Bug Fixes
+
+Paper fixes many vanilla bugs that were not intended by Mojang. These bugs are patched to fix behaviour or prevent abuse and
```
```suggestion
Paper fixes many vanilla bugs that were not intended by Mojang. These bugs are patched to fix behavior or prevent abuse and
```

Machine-Maker commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,53 @@
+
+## Duplication Bugs
+
+Paper Patches TNT, Carpet, Rail and Gravity Block (Sand, Gravel etc) Duplication bugs. These bugs are not intended and 
```
```suggestion
Paper Patches TNT, Carpet, Rail and Gravity Block (Sand, Gravel, etc.) Duplication bugs. These bugs are not intended and 
```

Machine-Maker commented on docs/paper/admin/misc/paper-bug-fixes.md:
```suggestion
If your server lags, crashes or has any other undesirable consequences of enabling these flags, you will not get support from Paper.
```
`of` I think is the usual preposition.

olijeffers0n replied:
Is it? Doesn't sound right to me

Machine-Maker commented on docs/paper/admin/misc/paper-bug-fixes.md:
```suggestion
These settings are also not guaranteed to be supported into the future and may have their behavior changed or removed at any time.
```

Machine-Maker reviewed (changes requested):
I think I would mention somewhere what our criteria for determine what is and isn't intended vanilla behavior. That is an open and confirmed issue on the tracker, usually with a mojang priority assigned to it.

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,64 @@
+
+# Paper Bug Fixes
+
+Paper fixes gameplay and mechanics inconsistencies. The most prevalent fixes being TNT Duplication and Bedrock Breaking.
```
```suggestion
Paper fixes gameplay and mechanic inconsistencies. The most prevalent fixes are to TNT duplication and bedrock breaking.
```

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,64 @@
+## Vanilla Bug Fixes
+
+Paper fixes many vanilla bugs that were not intended by Mojang. These bugs are patched to fix behavior or prevent abuse and
+instability on the server. Many of our fixes are configurable as we understand that some servers may want to keep the
```
```suggestion
instability on the server. Many of our fixes are configurable, as we understand that some servers may want to keep the
```

Machine-Maker replied:
I'm not sure we want to use the word `Many` here to describe the number of configurable fixes. I would say most aren't configurable. I think `Some` is better.

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,64 @@
+1) Has been confirmed as a bug
+2) Has an assigned priority to it
+
+If it meets these two criteria then we will accept changes to fix the bug as it can take a long time for Mojang to fix 
```
```suggestion
If it meets these two criteria, then we will accept changes to fix the bug, as it can take a long time for Mojang to fix 
```

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,64 @@
+2) Has an assigned priority to it
+
+If it meets these two criteria then we will accept changes to fix the bug as it can take a long time for Mojang to fix 
+them (Sometimes years). If an issue gets declined by Mojang, we normally do not "fix" it as it is intended behaviour.
```
```suggestion
them (sometimes years). If an issue gets declined by Mojang, we normally do not "fix" it as it is intended behaviour.
```

kennytv replied:
```suggestion
them (Sometimes years). If an issue gets declined by Mojang, we normally do not "fix" it, as it is intended behaviour.
```

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,64 @@
+
+## Duplication Bugs
+
+Paper Patches TNT, Carpet, Rail and Gravity Block (Sand, Gravel, etc.) Duplication bugs. These bugs are not intended and 
```
```suggestion
Paper patches TNT, carpet, rail and gravity block (sand, gravel, etc.) duplication bugs. These bugs are not intended and 
```

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```diff
@@ -0,0 +1,64 @@
+
+## Block Breaking
+
+We also fix the ability to break Bedrock and End Portal frames. We do also provide a config option to restore this
```
```suggestion
We also fix the ability to break bedrock and end portal frames. We do also provide a config option to restore this
```

Machine-Maker replied:
It's possible these are proper nouns as the name of blocks in game so capitalization would be appropriate.

kennytv commented on docs/paper/admin/misc/paper-bug-fixes.md:
```suggestion
We will not support you if you have issues whilst these settings are enabled, as they can cause unintended side effects.
These settings are also not guaranteed to be supported in the future and may have their behavior changed or removed at any time.

For reasoning behind not having configuration options for sand (and MANY other) duplication bugs, see: 
[#3724](https://github.com/PaperMC/Paper/issues/3724)

Please do not create any further discussion on config options for sand duping. The decision is final.
```

olijeffers0n commented:
Addressed Comments, some I did not use though as it seemed a bit trigger-happy with the commas and didn't seem correct (with a check online to back me up)
